app/views/chairs/show.html.erb

```diff
@@ -1,15 +1,16 @@
 <% content_for :title, "Chair #{@chair.name}" %>
 <div class='detail'>
-  <div class="row mb-5">
-    <div class="col-3">
-      <%= image_tag(@chair.image, class: "picture-rounded") %>
-    </div>
-    <div class="col">
-      <h1>
-        <%= @chair.name %>
-      </h1>
+  <div class='row mb-3'>
+    <div class='col-3'> <%= image_tag(@chair.image, class: "picture-rounded") %> </div>
+    <div class="col mx-3">
+      <h2><%= @chair.name %></h2>
+      <span class="badge bg-primary">Chair</span>
     </div>
   </div>
+  <div class="d-flex flex-row justify-content-start iconbuttons mb-3">
+    <%= link_to render('partials/iconbutton', :locals => {:text => 'Show on map', :icon_class => 'fa-map', :margin => 'me-4'}), map_path(:room_id => @chair.rooms[0]) %>
+    <%= link_to render('partials/iconbutton', :locals => {:text => 'Navigate', :icon_class => 'fa-location-arrow'}), root_path %>
```

Comment:
It looks a bit weird to use the root page as a placeholder but that should be okay if there is not better simple option

Reply:
Root_path seems to be the default placeholder on every show page so we also used it